c10/core/TensorImpl.h

```diff
@@ -33,6 +33,11 @@ C10_DECLARE_bool(caffe2_keep_on_shrink);
 // respect caffe2_keep_on_shrink.
 C10_DECLARE_int64(caffe2_max_keep_on_shrink_memory);
 
+// Error message to show when the user tries to change tensor metadata on
+// Tensor created from .data or .detach().
+//
+// For more detail, see comment for `allow_tensor_metadata_change_` field.
```

Comment:
we usually do this with [`NOTE ..]` so that it's easily greppable.

Reply:
Fixed